ui/app/AppLayouts/Profile/stores/ProfileStore.qml

```diff
@@ -15,8 +15,8 @@ QtObject {
     property string icon: !!Global.userProfile? Global.userProfile.icon : ""
     property bool userDeclinedBackupBanner: Global.appIsReady? localAccountSensitiveSettings.userDeclinedBackupBanner : false
     property var privacyStore: profileSectionModule.privacyModule
-    readonly property string keyUid: userProfile.keyUid
-    readonly property bool isKeycardUser: userProfile.isKeycardUser
+    readonly property string keyUid: !!Global.userProfile ? Global.userProfile.keyUid : ""
+    readonly property bool isKeycardUser: !!Global.userProfile ? Global.userProfile.isKeycardUser : false
```

Comment:
I remember this kid of bug. Are you sure that is not `undefined`?

Reply:
I added this just because of warnings in log. Do you think it should be undefined instead of `false`?

Reply:
No more warnings! 🙂